microsoft-edge/web-platform/release-notes/133.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Microsoft Edge 133 web platform release notes (Feb 2025)
+description: Microsoft Edge 133 web platform release notes (Feb 2025)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 01/31/2025
+---
+# Microsoft Edge 133 web platform release notes (Feb 2025)
+
+This article provides information about the new web platform features and updates in Microsoft Edge 133, which was released on February 6, 2025.
+
+To stay up to date and get the latest web platform features, download an Insiders preview version of Microsoft Edge. See [Microsoft Edge Insiders Channels](https://aka.ms/microsoftedge).
+
+<!-- ====================================================================== -->
+## Edge DevTools
+
+For Edge DevTools updates, see For Edge DevTools updates, see [What's New in Microsoft Edge DevTools](../../devtools-guide-chromium/whats-new/whats-new.md).
+
+
+<!-- ====================================================================== -->
+## WebView2
+
+For WebView2, see [Release Notes for the WebView2 SDK](../../webview2/release-notes/index.md).
+
+
+<!-- ====================================================================== -->
+## Web platform features
+
+
+<!-- ------------------------------ -->
+#### Enabled features
+
+###### Animation.overallProgress
+
+Adds an `overallProgress` property to the JavaScript class [Animation](https://developer.mozilla.org/docs/Web/API/Animation).
+
+This property provides authors with a convenient and consistent representation of how far along an animation has advanced across its iterations and regardless of the nature of its [timeline](https://developer.mozilla.org/docs/Web/API/AnimationTimeline).
+
+###### Atomics.pause
+
+Adds the `Atomics.pause` method to hint the CPU that the current code is executing a spinlock.
+
+###### CSP hash reporting for scripts
+
+Complex web application often need to keep tabs of the subresources that they download, for security purposes.
+
+In particular, upcoming industry standards and best practices (e.g. PCI-DSS v4) require that web applications keep an inventory of all the scripts they download and execute.
```

Comment:
@captainbrosset - link PCI-DSS v4?
I don't have a feel yet for what corrections and enhancements / conventions for this relnotes, given the pipeline.